### PR TITLE
add Conan 1.29 release blog post

### PR DESCRIPTION
**Pull request by solvingj**

### CODE REVIEW

File: _posts/2020-09-07-New-conan-release-1-29.markdown [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+On Windows, many users have reported an intermittant problem with python's
+`os.rename` function when used in some `conanfile.py` contexts. Sometimes the
+function will fail simply with a permission error. Numerous workarounds have
+been offerred, but in this release we've merged a new tools function called
+`tools.rename` which uses a more robust mechanism to avoid these errors.
+
+## Experimental Toolchain for Gnu Make
+
+Continuing our work on Conan's new toolchain strategy, this release contains a
+toolchain class for the `Gnu Make` build system. Like the existing `MSBuild` and
```
Comment (memsharded) [inline _posts/2020-09-07-New-conan-release-1-29.markdown:36]: MSBuild is not there yet. A new ``msbuild`` generator was added, but the toolchain is not there yet.

File: _posts/2020-09-07-New-conan-release-1-29.markdown [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+`CMake` toolchains which had been added previously, this new toolchain class
+generates `.mak` files for the `Gnu Make` build system which contain all the
+relevant build-related variables from Conan. To clarify, this does NOT include
+varaibles related to dependencies: this is still the domain of generators.
```
Comment (memsharded) [inline _posts/2020-09-07-New-conan-release-1-29.markdown:40]: varaibles => typo.

**Review by memsharded — APPROVED** [review]
Please fix those minor details, and it is ready to be published on Monday.
Thanks, great job!

File: _posts/2020-09-07-New-conan-release-1-29.markdown [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+Conan 1.29 has been released. This release doesn't contain any major new
+features, but instead brings more completeness and quality to existing
+features.
+
+## Updates to Generators
+
+Some of the newer `cpp_info` properties have now been added to both the `QMake`
+and `Qbs` generators, including `cpp_info.system_libs`,
+`cpp_info.framework_paths` and `cpp_info.frameworks`. Additionally, the `json`
+generator learned how to generate the `user_info` data.
```
Comment (jgsogo) [inline _posts/2020-09-07-New-conan-release-1-29.markdown:17]:
```suggestion
generator learned how to generate the `user_info` data from the build context.
```

File: _posts/2020-09-07-New-conan-release-1-29.markdown [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+relevant build-related variables from Conan. To clarify, this does NOT include
+varaibles related to dependencies: this is still the domain of generators.
+
+## Lockfile Bug Fixes
+
+We have also continued to evolve and improve the experience around Lockfiles. In
+this release, we fixed one bug where some package's were erroneously marked as
+modified, and another bug where `Package_ID_Unknown` were not being updated
+correctly.  We've also added a more helpful message when users try to compute
+the `build_order` of a graph from a "base" lockfile (which isn't possible.) Now
```
Comment (jgsogo) [inline _posts/2020-09-07-New-conan-release-1-29.markdown:48]:
```suggestion
the `build_order` of a graph from a "base" lockfile (which isn't possible). Now
```

**Review by jgsogo — APPROVED** [review]
👍  

Just a couple of details.

File: _posts/2020-09-07-New-conan-release-1-29.markdown [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+`os.rename` function when used in some `conanfile.py` contexts. Sometimes the
+function will fail simply with a permission error. Numerous workarounds have
+been offerred, but in this release we've merged a new tools function called
+`tools.rename` which uses a more robust mechanism to avoid these errors.
+
+## Experimental Toolchain for Gnu Make
+
+Continuing our work on Conan's new toolchain strategy, this release contains a
+toolchain class for the `Gnu Make` build system. Like the existing `MSBuild` and
+`CMake` toolchains which had been added previously, this new toolchain class
```
Comment (memsharded) [inline _posts/2020-09-07-New-conan-release-1-29.markdown:37]:
```suggestion
toolchain class for the `Gnu Make` build system. Like the existing `CMake` 
toolchain which had been added previously, this new toolchain class
```

**Review by memsharded — APPROVED** [review]

**Review by czoido — APPROVED** [review]

File: _posts/2020-09-07-New-conan-release-1-29.markdown [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+---
+
+Conan 1.29 has been released. This release doesn't contain any major new
+features, but instead brings more completeness and quality to existing
+features.
+
+## Updates to Generators
+
+Some of the newer `cpp_info` properties have now been added to both the `QMake`
+and `Qbs` generators, including `cpp_info.system_libs`,
```
Comment (danimtb) [inline _posts/2020-09-07-New-conan-release-1-29.markdown:15]: genrator names are always lower case. If you are talking about the build systems names, then disregard this suggestion

```suggestion
Some of the newer `cpp_info` properties have now been added to both the `qmake`
and `qbs` generators, including `cpp_info.system_libs`,
```

File: _posts/2020-09-07-New-conan-release-1-29.markdown [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+The `cmake_find_package` and `cmake_find_package_multi` both had some bugs in
+the previous release surrounding the new `filenames` attribute of `cpp_info`.
+Also, there was a bug in the `cmake_multi` generator in which it was producing
+`cmake` files which were failing under valid conditions. These have all been
+fixed.
+
+## New tools.rename Function
+
+On Windows, many users have reported an intermittant problem with python's
+`os.rename` function when used in some `conanfile.py` contexts. Sometimes the
```
Comment (danimtb) [inline _posts/2020-09-07-New-conan-release-1-29.markdown:28]: italics for folder, files or paths
```suggestion
`os.rename` function when used in some *conanfile.py* contexts. Sometimes the
```

File: _posts/2020-09-07-New-conan-release-1-29.markdown [hunk 8]
```diff
@@ -0,0 +1,60 @@
...
+function will fail simply with a permission error. Numerous workarounds have
+been offerred, but in this release we've merged a new tools function called
+`tools.rename` which uses a more robust mechanism to avoid these errors.
+
+## Experimental Toolchain for Gnu Make
+
+Continuing our work on Conan's new toolchain strategy, this release contains a
+toolchain class for the `Gnu Make` build system. Like the existing `CMake` 
+toolchain which had been added previously, this new toolchain class
+generates `.mak` files for the `Gnu Make` build system which contain all the
```
Comment (danimtb) [inline _posts/2020-09-07-New-conan-release-1-29.markdown:38]:
```suggestion
generates *.mak* files for the `Gnu Make` build system which contain all the
```

**Review by danimtb — APPROVED** [review]
Minor suggestions but looking good! 👍

**Comment by memsharded** [comment]
Sorry, @danimtb your review just got after the merge. Please open a PR to master if you want to integrate those improvements.

**Pull request merged**